src/services/spmNotification/index.test.ts

```diff
@@ -112,7 +112,23 @@ describe("single type oa-doc notification", () => {
     expect(notifyPdt).toBeCalledTimes(0);
   });
 
-  it("trigger notifyPdt for single valid SER test cert", async () => {
+  it("trigger notifyHealthCert for single valid SER test cert Staging Env", async () => {
+    process.env.STAGE = "stg";
```

Comment:
We should store the original value of `process.env.STAGE` and restore it afterwards.

```typescript
const ORIGINAL_STAGE = process.env.STAGE:
process.env.STAGE = "new value"

// Test something

process.env.STAGE = ORIGINAL_STAGE
```